app/main.py

```diff
@@ -1 +1,60 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
You need to extend `self.skills` instead of rewrite.
```suggestion
        self.skills += ["JavaScript", "HTML", "CSS"]
```